Lib/idlelib/editor.py

```diff
@@ -434,6 +435,11 @@ def set_line_and_column(self, event=None):
         line, column = self.text.index(INSERT).split('.')
         self.status_bar.set_label('column', 'Col: %s' % column)
         self.status_bar.set_label('line', 'Ln: %s' % line)
+        
+        selected = len(self.text.get(SEL_FIRST, SEL_LAST))
+        self.status_bar.set_label('selected',
+                                f"({selected} selected)" if selected else '',
+                                side=RIGHT)
```

Comment:
Is this required? I believe it to be redundant.

Reply:
I believe LEFT is defaut, so RIGHT is needed.  The following line have it also.

Reply:
I have not yet accepted the addition, let alone the format; further edits should await approval of both. If accepted, new tests will be needed.  Running tests before they are added is useless.

Reply:
@E-Paine  You were right.  I was looking at the first `set_label('selected', ...)`.  `side=Right` *is* redundant for subsequent calls as they only change the text of the existing label.  (A MultiStatusBar keeps a name-Label dict.)  Will omit in suggested rewrite below.